Patches/AnimalCollabProj/AnimalCollabProj_CE_Patch_Races_Animal_Silkie.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationSequence">
+		<success>Always</success>
+		<operations>
+
+			<li Class="CombatExtended.PatchOperationFindMod">
+				<modName>AnimalCollabProj</modName>
+			</li>
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="ACPSilkieChicken"]/statBases</xpath>
+				<value>
+					<MeleeDodgeChance>0.15</MeleeDodgeChance>
+					<MeleeCritChance>0</MeleeCritChance>
+					<MeleeParryChance>0</MeleeParryChance>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="ACPSilkieChicken"]/tools</xpath>
```

Comment:
No claw scratch attack like the vanilla chicken?

Reply:
Yeah, just realized Silkies are otherwise identical to vanilla chickens.

Fixed in 46adb9e